### ISSUE TITLE
please update the documentation and online-help

### ISSUE THREAD

**Issue by Ventusfahrer** [btittelbach/openmtbmap_openvelomap_linux#20]
According to the link https://www.velomap.org/updates/mapinstall-finally-fixed-by-garmin-download-lists-reworked-gmapsupp-img-downloads-now-showing-in-basecamp-by-default/ (the 2nd last paragraph):
`Lastly - the windows installer for Germany finally broke the 2GB limit. The installers for France, Africa continent and Russia were also close to breaking the 2GB limit. I don't want to offer these countries as 2 downloads - so I've updated the installer to accept sizes over 2GB. However this causes the installer not to be extractable anymore with 7zip. Therefore I am now creating a new zipped version of those countries for Linux users. Also of course some bugs in the map creation got fixed.`

If the .exe file is bigger than 2 GB, velomap.org ist offering a second download file which hyperlink is named **_Velomap - \<country> - Linux_**. The link points to a **_velomap\_\<country>.7z_** file which has to be used as argument to create_omtb_garmin_img.sh

in other words:
if you have a .exe smaller than 2 GB then you call:
   create_omtb_garmin_img.sh [options] <mtb*.**exe**|velo*.**exe**> <TYP-file or TYP-style>
otherwise you call
   create_omtb_garmin_img.sh [options] <mtb*.**7z**|velo*.**7z**> <TYP-file or TYP-style>
